Cargo.toml

```diff
@@ -157,6 +157,7 @@ crossterm = { workspace = true }
 ctrlc = { workspace = true }
 dialoguer = { workspace = true }
 flate2 = { workspace = true }
+graphql_client = { workspace = true }
```

Comment:
all of our `graphql_client` code typically lives in the `rover-client` workspace crate. this endpoint will likely be in studio and so the boilerplate we have defined there should work great for this once it's part of our own infrastructure.

Reply:
I'm working on tracking down how to get that added. Once we do I'll reorganize this so we can reuse the same client. I should be able to mix and match generated and macro queries... I hope.